src/background/service/ExtDappService.js

```diff
@@ -199,10 +199,10 @@ class ExtDappService {
         let message = params.message
         let address = params.address
 
-        await this.dappOpenPopWindow('./popup.html#/request_sign?isUnlocked=' + isUnlocked
-          + "&siteUrl=" + siteUrl
-          + "&context=" + context + "&message=" + message + "&address=" + address
-          + "&siteIcon=" + icon,
+        await this.dappOpenPopWindow('./popup.html#/request_sign?isUnlocked=' + encodeURIComponent(isUnlocked)
+          + "&siteUrl=" + encodeURIComponent(siteUrl)
+          + "&context=" + encodeURIComponent(context) + "&message=" + encodeURIComponent(message) + "&address=" + encodeURIComponent(address)
+          + "&siteIcon=" + encodeURIComponent(icon),
```

Comment:
A nicer alternative
```
'./popup.html#/request_sign?' + new URLSearchParams({isUnlocked, siteUrl, context, message, address, siteIcon: icon}).toString()
```

Reply:
thanks, I'll open a request for enhancement to do it that way. there are several other places where we build params strings that this PR won't touch

Reply:
> thanks, I'll open a request for enhancement to do it that way. there are several other places where we build params strings that this PR won't touch

Thanks, @pro-wh!

Reply:
https://github.com/oasisprotocol/oasis-wallet-ext/issues/78 issue